http-core/src/main/scala/org/apache/pekko/http/javadsl/settings/ServerSettings.scala

```diff
@@ -71,7 +70,7 @@ import scala.concurrent.duration.{ Duration, FiniteDuration }
   // ---
 
   def withServerHeader(newValue: Optional[Server]): ServerSettings =
-    self.copy(serverHeader = newValue.asScala.map(_.asScala))
```

Comment:
The `map` here was entirely redundant as it seemed to be calling an implicit `asScala` that just returned itself